### PR TITLE
Package for dbvisualizer 9.2.5

### PR DESCRIPTION
**Pull request by fooblahblah**

### CODE REVIEW

File: pkgs/applications/misc/dbvisualizer/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+stdenv.mkDerivation {
+  name = "dbvisualizer-9.2.5";
+
+  src = fetchurl {
+    url = https://www.dbvis.com/product_download/dbvis-9.2.5/media/dbvis_unix_9_2_5.tar.gz;
+    sha256 = "e9d3374e96cf63903ebd9dae498c6b69403813eb8723187ed70f285dd7119327";
+  };
+
+  phases = "unpackPhase installPhase";
```
Comment (lethalman) [inline pkgs/applications/misc/dbvisualizer/default.nix]: You don't need this line.

File: pkgs/applications/misc/dbvisualizer/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    mkdir -p $out/bin
+    cp -a . $out
+    ln -sf $out/dbvis $out/bin    
+    wrapProgram $out/bin/dbvis --set INSTALL4J_JAVA_HOME ${jre}
+  '';
+  
+  meta = {
+    description = "The universal database tool";
+    homepage = https://www.dbvis.com/;
+    license = "unfree";
```
Comment (domenkozar) [inline pkgs/applications/misc/dbvisualizer/default.nix]: `stdenv.lib.licenses.unfree`

**Comment by domenkozar** [comment]
Two small issues and it's ready. Thanks!

**Comment by fooblahblah** [comment]
Thanks I'll make those changes. Actually I should double check that license
anyway...

**Comment by lethalman** [comment]
When you commit to a PR we don't get notified, please always add a comment when you want a further review.

**Comment by lethalman** [comment]
Looks fine, thanks.

**Comment by fooblahblah** [comment]
Thanks, will do.

-Jeff

**Pull request merged**